ruma-events/src/key/verification/start.rs

```diff
@@ -40,7 +41,7 @@ pub struct MSasV1Content {
 
     /// The key agreement protocols the sending device understands.
     ///
-    /// Must include at least `curve25519`.
+    /// Must include at least `Curve25519` or `Curve25519HkdfSha256`.
```

Comment:
While we're stricly speaking not compatible with either version of the spec like this, I really like this solution.